Sources/Core/Cart/ShoppingCartProviding.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ShoppingCartProviding.swift
+//  
+//
+//  Created by Uwe Tilemann on 21.12.22.
+//
+
+import Foundation
+
+public protocol ShoppingCartMerging: AnyObject {
```

Comment:
Aus welchem Grund hast du das nicht ans `ShoppingCartDelegate` gehangen, bzw. an den Workaround, da das `ShoppingCartDelegate` kein Delegate ist. Daher eher an das `InternalShoppingCartDelegate?

Reply:
ShoppingCart und die Funktion `add(_:)` bzw. `quantity(of:)` ist im Core. Das sind die Stellen über die das Morgen entschieden wird. Im Core gibt es zwar den `InternalShoppingCarDelegate` und der ist public deklariert wird aber nur SDK intern genutzt.